controller/controller-board/src/app/components/services/taskService.js

```diff
@@ -13,14 +13,16 @@
             getTasks : function(namespace, clustername, state) {
                 var url = 'https://controllerhttp.pinadmin.com/v1/tasks?' +
                     (namespace === 'UNDEFINED' ? '': 'namespace=' + namespace + '&')  +
-                    (clustername === 'UNDEFINED' ? '': 'clustername=' + clustername + '&')  +
+                    (clustername === 'UNDEFINED' ? '': 'clusterName=' + clustername + '&')  +
                     'state=' + state;
                 return $http.get(url);
-            }
+            },
+
+            replaceHost : function (namespace, clusterName, oldHost, newHost) {
+                var url = 'https://controllerhttp.pinadmin.com/v1/clusters/replaceHost/'
+                    + namespace + '/'+ clusterName + '?oldHost=' + oldHost + '&newHost=' + newHost + '&force=True';
```

Comment:
for replace host, it also supports not giving a newHost, controller will pick the host automatically. we can add this support later on another diff